webrender/src/device/gfx/device.rs

```diff
@@ -479,21 +479,19 @@ impl<B: hal::Backend> Device<B> {
         let pipeline_requirements: FastHashMap<String, PipelineRequirements> =
             from_str(&shader_source::PIPELINES).expect("Failed to load pipeline requirements");
 
-        let mut descriptor_pools_per_draw = SmallVec::new();
+        let descriptor_pools_per_draw = DescriptorPools::new(
```

Comment:
I find it confusing that it's called `DescriptorPools` in plural. Could it be renamed to just `DescriptorPool`, or would that be wrong?